go-controller/pkg/node/gateway_localnet.go

```diff
@@ -342,7 +342,6 @@ func (l *localPortWatcher) SyncServices(serviceInterface []interface{}) {
 	}
 	for _, chain := range []string{iptableNodePortChain, iptableExternalIPChain} {
 		recreateIPTRules("nat", chain, keepIPTRules)
-		recreateIPTRules("filter", chain, keepIPTRules)
```

Comment:
@alexanderConstantinescu should we handle on upgrade removing these rules?

Reply:
Ah man, you're completely right. I overlooked that. Let me try to add something which lists all existing rules and remove the ones that are not supposed to exist.